Formula/psqlodbc.rb

```diff
@@ -19,13 +19,19 @@ class Psqlodbc < Formula
   end
 
   depends_on "openssl"
-  depends_on "unixodbc"
   depends_on :postgresql
+  depends_on "libiodbc" => :optional
+  depends_on "unixodbc" if build.without? "libiodbc"
```

Comment:
Just make this `depends_on "unixodbc" => :recommended`. It's not as smooth in terms of avoiding the dependency but currently Homebrew handles `if build.with?` style conditionals outside of `def install` really quite awfully.